src/main/java/org/fogbowcloud/saps/engine/core/database/Catalog.java

```diff
@@ -31,14 +30,8 @@ SapsImage addImageTask(String taskId, String dataset, String region, Date date,
 	void addUser(String userEmail, String userName, String userPass, boolean userState, boolean userNotify,
```

Comment:
That name `addStateStamp` doesn't look good to me. Would something like `setStateChangeTime` be better?

Reply:
setStateChange seems to me that there will be an update somewhere, but in fact it is an insertion of new information in the Catalog about a change of state of a task.

Reply:
So could you replace it with `addStateChangeTime`?

Reply:
lgtm @wesleymonte, done